db/crud.go

```diff
@@ -1163,20 +1163,14 @@ func (db *DatabaseCollectionWithUser) resolveConflict(ctx context.Context, local
 	// TODO: Make doc expiry (_exp) available over replication.
 	// remoteExpiry := remoteDoc.Expiry
 
-	localDocBody, err := localDoc.GetDeepMutableBody()
-	if err != nil {
-		return "", nil, err
-	}
+	localDocBody := localDoc.Body().ShallowCopy()
```

Comment:
This needs to be `GetDeepMutableBody()` for otto and probably `Body().ShallowCopy()` for V8 - this is enough that I'd be tempted to write a shim to get the right part of the body?

I think not interacting with otto as go objects directly has the potential to slow down otto, which I don't think we want to do.